erpnext/public/js/controllers/transaction.js

```diff
@@ -1910,10 +1910,7 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 
 	set_query_for_item_tax_template: function(doc, cdt, cdn) {
 		var item = frappe.get_doc(cdt, cdn);
-		if(!item.item_code) {
-			frappe.throw(__("Please enter Item Code to get item taxes"));
-		} else {
-
+		if(item.item_code) {
```

Comment:
@casesolved-co-uk please filter item tax templates by company if item code is not entered. Currently, it won't apply any filter in case item code is not entered.

Reply:
Now done and tested